### PR TITLE
Add random velocity for drops

### PR DESCRIPTION
**Pull request by LuceCarter**
**Changes Made:**

* Added random numbers for min and max to the velocity property for each type of drop command  (respecting the previous desired min and max values)
* Added the additional ID's for my other two emotes

**Comments:**

This may not be the best way to achieve the random velocities but I wasn't sure. I know you review all PR's so this is selfishly a way of getting a free code review so if it could be done better,  I can learn from it 👍

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -17,49 +17,49 @@ export default {
     '!drop': {
       emoteMultiplier: 1,
       velocities: {
-        min: 3,
-        max: 7,
+        min: Math.floor(Math.random() * 5) + 3,
```
Comment (MadhouseSteve) [inline src/config.js:20]: The min here is between 3 and 8
The max here is between 5 and 12

So a chance that your max could be lower than your min.

File: src/config.js [hunk 2]
```diff
@@ -17,49 +17,49 @@ export default {
...
       },
       strategy: 'dropSpecificSizedPanthers',
       size: emotes.sizes[1],
     },
     '!blizzard': {
       emoteMultiplier: 15,
       velocities: {
-        min: 20,
-        max: 30,
+        min: Math.floor(Math.random() * 25) + 20,
```
Comment (MadhouseSteve) [inline src/config.js:45]: The min here is between 20 and 45
The max here is between 25 and 55

So a chance that your max could be lower than your min.

**Review by MadhouseSteve — CHANGES_REQUESTED** [review]
You need to add some checks to make sure your max is greater than your min. Otherwise looks good to me.

**Comment by whitep4nth3r** [comment]
As a result of running Math.random in the config file, the file loads when the application starts with a fixed random velocity, and doesn't rerun every time the commands are run.

It might be a better idea to put the maths in Drop.js as that runs every time a command is generated and will therefore generate a random number.

Thank you so much for your contribution!

**Pull request merged**